src/sidebar/services/groups.js

```diff
@@ -27,6 +27,7 @@ function groups(
   serviceUrl,
   session,
   settings,
+  auth,
   features
```

Comment:
I forgot to flag this up before, but longer injected parameter lists (functions with `@ngInject` above them) are easiest to work with if kept sorted.